docs/standards/smart-contracts/lsp0-erc725-account.md

```diff
@@ -268,6 +278,27 @@ _Triggers the **[OwnershipTransferred](#ownershiptransferred)** event after succ
 Leaves the contract without an owner. Once ownership of the contract is renounced, it won't be possible to call the functions restricted to the owner only.
 :::
 
+### batchCalls
+
+```solidity
+function batchCalls(bytes[] calldata data) public returns (bytes[] memory results)
+```
+
+Allows a caller to batch different function calls in one call. Performs a delegatecall on self to call different functions while preserving the context. It is not possible to send value along the functions call due to the use of delegatecall.
```

Comment:
```suggestion
Allows a caller to batch different function calls in one call. Performs a `delegatecall` on self to call different functions while preserving the context. It is impossible to send value along the functions call due to the use of delegatecall.
```

I would suggest either using "delegate call" within the text or using code brackets for the function name